adapters/repos/db/search.go

```diff
@@ -181,6 +181,10 @@ func (d *DB) objectSearch(ctx context.Context, offset, limit int,
 	additional additional.Properties) (search.Results, error) {
 	var found []*storobj.Object
 
+	if err := d.validateSort(sort); err != nil {
+		return nil, errors.Wrap(err, "search")
```

Comment:
Thinking that a more detailed error message would be useful here, as the errors returned by `d.validateSort` don't provide a lot of context either. 

For example, let's say that `d.validateSort` returns errors.  This would leave us something like:

```
"search index 123, failed to do this, search index 456, failed to do that"
```

then once `db.objectSearch` returns, we end up with:
```
"search: search index 123, failed to do this, search index 456, failed to do that"
```

what I'm suggesting is maybe replacing `errors.Wrap(err, "search")`  with at least `errors.Wrap(err, "objectSearch")` or similar, so that it is immediately apparent what went wrong when looking at error logs.

Additionally, maybe would be good to put something about sort validation failing when `validateSort` errors out, rather than just `search index ...`